cmd/probe/internal/component/postgres/manager.go

```diff
@@ -858,9 +846,33 @@ func (mgr *Manager) CreateRoot(ctx context.Context) error {
 }
 
 func (mgr *Manager) Lock(ctx context.Context, reason string) error {
-	return nil
+	mgr.Logger.Infof("Lock db: %s", reason)
```

Comment:
logging with the result ( and possible error)

Reply:
done